cookbooks/bcpc-hadoop/recipes/oozie.rb

```diff
@@ -19,7 +19,7 @@
 
 OOZIE_LIB_PATH='/usr/hdp/current/oozie'
 OOZIE_CLIENT_PATH='/usr/hdp/current/oozie-client'
-OOZIE_SERVER_PATH='/usr/hdp/2.2.0.0-2041/oozie-server'
+OOZIE_SERVER_PATH='/usr/hdp/2.2.0.0-2041/oozie/oozie-server'
```

Comment:
Can we use `/usr/hdp/current/oozie-server/oozie-server` instead of hard coding the `HDP` version?